packages/workspace/src/command-line/nx-commands.ts

```diff
@@ -231,41 +232,42 @@ function withPrintAffectedOptions(yargs: yargs.Argv): yargs.Argv {
 
 function withPlainOption(yargs: yargs.Argv): yargs.Argv {
   return yargs.option('plain', {
-    describe: 'Produces a plain output for affected:apps and affected:libs',
+    describe:
+      'Produces a plain output for `affected:apps` and `affected:libs`.',
   });
 }
 
 function withAffectedOptions(yargs: yargs.Argv): yargs.Argv {
   return yargs
     .option('files', {
       describe:
-        'Change the way Nx is calculating the affected command by providing directly changed files, list of files delimited by commas',
+        'Change the way Nx is calculating the `affected` command by providing directly changed files, list of files delimited by commas.',
       type: 'array',
       requiresArg: true,
       coerce: parseCSV,
     })
     .option('uncommitted', {
-      describe: 'Uncommitted changes',
+      describe: 'Uncommitted changes.',
```

Comment:
Can we undo the additions of the `.` in this PR?

I'm not sure all of the descriptions (such as this one) constitute a full sentence which is why they don't have periods now.

I think separating these changes are going to make them easier to review.

Reply:
Yeah, I'll have this in a separate PR.

Regarding the periods, in my opinion, they should always be there, it doesn't make sense to have some descriptions with them and others without them. If anything, this only shows that those descriptions are not as descriptive as they should be, that's the real issue, but every description should end with a period.